tools/pagerduty-migrator/README.md

```diff
@@ -80,6 +80,7 @@ docker run --rm \
 -e ONCALL_API_URL="<ONCALL_API_URL>" \
 -e ONCALL_API_TOKEN="<ONCALL_API_TOKEN>" \
 -e ONCALL_DEFAULT_CONTACT_METHOD="sms" \
+-e FALLBACK_INTEGRATION_TYPE="webhook" \
```

Comment:
We probably could rename this to `ONCALL_DEFAULT_INTEGRATION_TYPE` similar to `ONCALL_DEFAULT_CONTACT_METHOD` so it's more consistent

Reply:
change applied: https://github.com/grafana/oncall/pull/1015/commits/6e26085464cd7b4d02b14a0186ac1456d1ca4520